tox.ini

```diff
@@ -0,0 +1,61 @@
+[tox]
+envlist =
+    py{36,37,38,39}-test{,-alldeps,-oldestdeps}{,-devastropy}{,-cov}
+    codestyle
+    build_docs
+requires =
+    setuptools >= 30.3.0
+    pip >= 19.3.1
+    tox-pypi-filter >= 0.12
+isolated_build = true
+
+
+[testenv]
+
+# The following option combined with the use of the tox-pypi-filter above allows
+# project-wide pinning of dependencies, e.g. if new versions of pytest do not
+# work correctly with pytest-astropy plugins. Most of the time the pinnings file
+# should be empty.
+pypi_filter = https://raw.githubusercontent.com/astropy/ci-helpers/master/pip_pinnings.txt
+
+# Run the tests in a temporary directory to make sure that we don't import
+# astropy from the source tree
+changedir = .tmp/{envname}
+
+description = run tests
+
+deps =
+    devastropy: git+https://github.com/astropy/astropy.git#egg=astropy
```

Comment:
I don't understand why the job with devastropy is also building pyregion from source.

```
 ERROR: Failed building wheel for pyregion
```

Reply:
Ah!!! It is a Python 3.9 job. Perhaps pyregion doesn't have wheel for it.

Reply:
> Do you know a trick of how to remove a dependency from the all list?

Unfortunately, I don't. The only options I know of are either refactoring the listing in `setup.cfg` or manually state your dependencies in `tox.ini`.